monai/inference_instructions.md

```diff
@@ -0,0 +1,75 @@
+## Instructions for running inference with the contrast-agnostic spinal cord segmentation model
+
+The following steps are required for using the contrast-agnostic model. 
+
+### Setting up the environment and Installing dependencies
+
+The following commands show how to set up the environment. Note that the documentation assumes that the user has `conda` installed on their system. Instructions on installing `conda` can be found [here](https://conda.io/projects/conda/en/latest/user-guide/install/index.html).
+
+1. Create a conda environment with the following command:
+
+```bash
+conda create -n venv_monai python=3.9
+```
+
+2. Activate the environment with the following command:
+
+```bash
+conda activate venv_monai
+```
+
+3. The list of necessary packages can be found in `requirements_inference.txt`. Use the following command for installation:
+
+```bash
+pip install -r requirements_inference.txt
+```
+
+### Method 1: Running inference on a single image
+
+```bash
+python run_inference_single_image.py --path-img /path/to/my-awesome-SC-image.nii.gz --chkp-path /path/to/best/model --path-out /path/to/output/folder --crop-size <64x160x320> --device <cpu/gpu>
+```
+    
+`--path-img` - Path to the image to be segmented
+`--chkp-path` - Path to the model checkpoint. This folder should contain the `best_model_loss.ckpt`
```

Comment:
where can we find this? pls include a temporary path to the latest model so we can test it-- until we upload as a release asset

Reply:
I mentioned that path to the model here  https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/pull/60#issuecomment-1753057589